### PR TITLE
[nfc][cxx-interop] Add some tests for calling C++ methods.

### PR DESCRIPTION
**Pull request by zoecarver**
Finally 😅

### CODE REVIEW

**Comment by zoecarver** [comment]
@swift-ci please smoke test.

**Comment by zoecarver** [comment]
@swift-ci please smoke test.

File: test/Interop/Cxx/class/method/Inputs/methods.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  ~NonTrivialInWrapper() { }
+};
+
+struct HasMethods {
+  void nonConstMethod() { }
+  void constMethod() const { }
+
+  int nonConstPassThrough(int a) { return a; }
+  int constPassThrough(int a) const { return a; }
```
Comment (compnerd) [inline test/Interop/Cxx/class/method/Inputs/methods.h:15]: Not sure what "passthrough" refers to here.  Is it the argument or the fact that they are inline?

File: test/Interop/Cxx/class/method/Inputs/module.modulemap [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module Methods {
+  header "methods.h"
+  requires cplusplus
+}
```
Comment (compnerd) [inline test/Interop/Cxx/class/method/Inputs/module.modulemap:4]: Missing newline

**Review by compnerd — APPROVED** [review]

**Comment by zoecarver** [comment]
@swift-ci please smoke test.

File: test/Interop/Cxx/class/method/Inputs/methods.h (hunk 1 above)
Comment (zoecarver) [reply test/Interop/Cxx/class/method/Inputs/methods.h:15]: Yeah, just passing through the arg.

**Comment by zoecarver** [comment]
Thanks, Saleem!

**Pull request merged**